runtime/parser2/expression.go

```diff
@@ -561,6 +562,21 @@ func defineReferenceExpression() {
 	)
 }
 
+func defineMemberExpression() {
+	setExprLeftBindingPower(lexer.TokenDot, 140)
+	setExprLeftDenotation(
+		lexer.TokenDot,
+		func(p *parser, token lexer.Token, left ast.Expression) ast.Expression {
+			p.skipSpaceAndComments(true)
+			identifierToken := p.mustOne(lexer.TokenIdentifier)
+			return &ast.MemberExpression{
+				Expression: left,
+				Identifier: tokenToIdentifier(identifierToken),
+			}
```

Comment:
The existing `mustIdentifier` helper can be used here:

```suggestion
			identifier := mustIdentifier(p)
			return &ast.MemberExpression{
				Expression: left,
				Identifier: identifier,
			}
```